cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -115,7 +115,7 @@ function is_taint_absence {
 function wait_for_taint_absence {
   local -r taint=$1
 
-  local -r tries=24
+  local -r tries=30
```

Comment:
Continuing the root thread here

> @oshoval
> For all the jobs that were in failure status it was due to a test that fail due to etcdserver timeout error or
> failure on creating CSR or a Secret due to etcdserver timeout error.
> The only job that was failed due to the not waiting enough for taints was on the PR that presented at
> kubevirt/kubevirt#4545 job
> and a draft PR to reflect #469 on kubevirt job
> So we are good.

Not in my opinion, taking 10 seconds margins when the worst case is 140 is prone for mistakes, 7% is too little, and 2 days is too short period to test stuff.
It also can combined with what Edy suggested, having the margins, especially in the beginning for collecting more statistics and be on the safe side, is blessed.

> >  you can also put a dynamic taint TO
> > based on the hardware type, based on the PR description

> What do you mean?

There are couple types of hardware, each has own characteristics about the needed timeout counters,
in case desired, you can detect the hardware dynamically and set the timeouts according that.

Reply:
I monitored how much it takes for the job to finish for few weeks like I said on other conversations and not two days
so I got the margins, that 140 happen one time only
We can always increase the time if we will need to.

Its not caused directly by the hardware its how sriov-operator works and put the taints, 
I think its overkill, at least for now, to reverse engineer all the possibilities for the sriov operator to plane taints

Reply:
My comments stands still please, its not overkill, its to have margins that are needed and that dont cost anything (see link below).
Also Edy said it as well imo here, in little different words but same result.
https://github.com/kubevirt/kubevirtci/pull/477#issuecomment-726245408

Moreover 
https://github.com/kubevirt/kubevirtci/pull/477#issuecomment-726627678
Make it even stronger, since the logic should be fixed as Edy and myself raised (see please my comment inline there).

Reply:
There is no tumb rule about how much "grace time" we need to set if 7% 
is little how much do you suggests?

Reply:
Lets first be on-sync on what Edy and myself raised please (see links above, especially the inline comment of https://github.com/kubevirt/kubevirtci/pull/477#issuecomment-726627678)
because it will affect the answer to this question.